rest-api-spec/src/main/resources/rest-api-spec/api/ml.get_trained_models.json

```diff
@@ -43,13 +43,6 @@
         "required":false,
         "description":"A comma-separate list of fields to optionally include. Valid options are 'definition' and 'total_feature_importance'. Default is none."
       },
-      "include_model_definition":{
-        "type":"boolean",
-        "required":false,
-        "description":"Should the full model definition be included in the results. These definitions can be large. So be cautious when including them. Defaults to false.",
-        "default":false,
-        "deprecated": true
-      },
```

Comment:
Why is this deleted?

Reply:
Ah the code was removed in 9.0 by this PR https://github.com/elastic/elasticsearch/pull/118103